src/travel-search-filters/FiltersContext.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {createContext, useContext, useEffect, useState} from 'react';
+import {storedFilters} from './storage';
+import {TransportModeFilterOptionWithSelectionType} from '@atb/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/types';
+
+type FiltersContextState = {
+  filters: TransportModeFilterOptionWithSelectionType[];
+  setFilters(
+    filters: TransportModeFilterOptionWithSelectionType[] | undefined,
+  ): Promise<void>;
```

Comment:
I see that where this is used we don't await the promise. Since the caller doesn't care or want to know that this returns a `Promise`, maybe we can just make it a void-function?